### DISCUSSION TITLE
pyOpenSci Reviews Scientific Python Software

### DISCUSSION THREAD

**Discussion by tkoyama010** [bjlittle/geovista#1063] [Ideas]
Are you interested in about peer review of scientific Python software?

<h1 align="center">
  <a href="https://www.pyopensci.org/about-peer-review/index.html">
    <img src="https://www.pyopensci.org/software-peer-review/_static/logo-light-mode.png"
         alt="pyopensci"
         width="200"></a>
  <div>peer review of scientific Python software</div>
</h1>

**Comment by trexfeathers** [comment]
We've asked pyOpenSci questions at conferences before and it seems to be aimed at scientists who need software help. @bjlittle is a rockstar, and it looks like you are too @tkoyama010 ⭐. I'd personally rather give pyOpenSci more time with people who need help, but I could be persuaded!

**Reply by bjlittle** [reply]
@tkoyama010 Thanks! 

When I saw pyOpenSci advertised at SciPy 2024 I wondered whether that might be an interesting route to take, particularly because once you've been through their workflow it apparently expedites the final review process with JOSS. But you've got experience with publishing a paper with JOSS, right? 

If so, then that might be enough ... Or was there something that you wanted to explore by using pyOpenSci?

**Reply by tkoyama010** [reply]
No, I was not a PyVista project member in 2019 when PyVista submitted the paper to JOSS, so I do not know the details of the process there. I am now creating a new project [scikit-gmsh](https://github.com/pyvista/scikit-gmsh) and intend to use that project to experience using pyOpenSci and JOSS once. After that experience I will consider if it would be beneficial to GeoVista :)

**Reply by bjlittle** [reply]
@tkoyama010 Awesome 👌

BTW have you seen:
- https://github.com/meshpro/pygalmesh
- https://github.com/meshpro/optimesh
- https://github.com/nschloe/pygmsh

`pygmsh` seems pretty similar to `scikit-gmsh`.

I've been watching those repos for a while as I'm interested in remeshing algorithms for some functionality required by `geovista`.

Although, `scikit-gmsh` looks close to what I need too ... Awesome, I'll take a deeper look at it 🚀

**Reply by tkoyama010** [reply]
Yes. I am a big fan of them and have used them for years😊I created scikit-gmsh to use it more sequentially with PyVista.

**Reply by bjlittle** [reply]
Great! 

You've also answered my next question. I was guessing that `scikit-gmsh` was going to provide an integrated API between `gmsh` and `pyvista`.

If that's the case then I'm now super interested in `scikit-gmsh`! 🤩

**Comment by bjlittle** [comment]
@tkoyama010 As an aside, do you have experience of what's involved with getting a project affiliated with NumFOCUS?

I'd so, what are the advantages of doing that?

Just curious 🤔

**Reply by tkoyama010** [reply]
If you project are an OSS for science, you are free to apply from [Affiliated Projects](https://numfocus.org/sponsored-projects/affiliated-projects). Requirements are checked when you apply, so please try first 😀
